contracts/src/systems/actions.cairo

```diff
@@ -170,6 +171,20 @@ mod actions {
         fn test_func(world: IWorldDispatcher, game_id: u128) -> bool{
             let _mancala_game: MancalaGame = get!(world, game_id, (MancalaGame));
             true
+        }
+
+             fn restart_game(
```

Comment:
In order for a game to restart both players must confirm that they are accepting a restart. Does this logic exist in your updates? If not please implement. A suggested approach would be to store a field on the GamePlayer model restart_requested: bool. And if both players have restart_requested=true then perform the restart logic

Reply:
you were not specific about how the restart function should be from start. besides the current merge you made cannot build.